pom.xml

```diff
@@ -337,8 +337,14 @@
         <dependency>
             <groupId>org.neo4j</groupId>
             <artifactId>neo4j-graphviz</artifactId>
-            <version>3.1.1</version>
-            <scope>provided</scope>
+	    <version>3.1.1</version>
+	    <scope>provided</scope>
+            <exclusions>
+                <exclusion>
```

Comment:
I've seen that all the neo4j 3.1.1 dependencies are pulled otherwise. The dependency is required since neo4j-graphviz has been removed from the product in 3.2.x as far as I know